blockauth/apple/tests/test_web_views.py

```
@@ -224,6 +224,66 @@ def test_callback_clears_cookies_on_error(apple_settings, client):
+# ---------------------------------------------------------------------------
+#
+# Re-usable error-path cleanup so BFF integrators that swap the response
+# shape (and therefore bypass AppleWebCallbackView.handle_exception) don't
+# have to re-implement the security-critical state/PKCE/nonce clear in
+# every consumer. Owning "what cookies need clearing on Apple's error
+# path" in blockauth keeps the contract in one place: if Apple's auth
+# flow ever grows another cookie, this helper picks it up and every
+# integrator that calls it stays correct.
+
+
+def test_clear_apple_callback_cookies_clears_all_three_cookies(apple_settings):
```

Ruff (0.15.12)

[warning] Unused function argument: `apple_settings`

(ARG001)